schemas/recipes_schema.json

```diff
@@ -2,11 +2,10 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "recipes",
   "description":"A dictionary of quoted numerical item IDs. Each ID maps to a list of recipes. There may be multiple different recipes per item (same ID and metadata). The recipes may not be sorted.",
-
   "definitions": {
     "id": {
       "type":["integer","null"],
-      "description":"A single numerical ID or null."
+      "description":"A single numerical ID, text id or null."
```

Comment:
Remove that from the schema now that IDs are all int

Reply:
Done